docker-compose/docker-compose.yml

```diff
@@ -0,0 +1,52 @@
+version: '3'
+
+networks:
+  monitor-net:
+    driver: bridge
+
+services:
+
+  surveyor:
+    image: synadia/nats-surveyor
```

Comment:
should we set a tag? perhaps via a variable? so its easy to upgrade else people have to know how to juggle docker and rm old images etc to get it to update